packages/joint-core/src/shapes/standard.mjs

```diff
@@ -98,16 +98,16 @@ export const Ellipse = Element.define('standard.Ellipse', {
 export const Path = Element.define('standard.Path', {
     attrs: {
         body: {
-            refD: 'M 0 0 L 10 0 10 10 0 10 Z',
+            d: 'M 0 0 H calc(w) V calc(h) H 0 Z',
```

Comment:
Shouldn't we be using normalized path data? i.e. `L` with four calc() expressions

Reply:
We are not doing normalizations on the data later, so the solution with V and H (and two calc expressions) is better than an alternative solution with L (and four calc expressions)